packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -82,18 +85,31 @@ function MaybeIframe( {
 	);
 }
 
+function findPostContent( blocks ) {
+	for ( let i = 0; i < blocks.length; i++ ) {
+		if ( blocks[ i ].name === 'core/post-content' ) {
+			return blocks[ i ];
+		}
+		if ( blocks[ i ].innerBlocks.length ) {
+			return findPostContent( blocks[ i ].innerBlocks );
```

Comment:
I _think_ we only want to `return` here if we've found a post content block? Otherwise it looks like `findPostContent` bails early without having found something if the first depth-first discovery does not match a post content block.

In my current template, it bails after examining the deepest separator block, rather than moving up a level to eventually find the post content block.

In the below screenshot, I logged out `looking at` for each instance of the for loop, and it appears to stop at the `core/separator`, where it should have moved up a level and found `core/spacer` and then moved down one to `core/post-content`.

<img width="1113" alt="image" src="https://user-images.githubusercontent.com/14988353/191151946-856009d5-c173-4d19-98c9-913cacf54555.png">

Reply:
Ahhhh well spotted! Yeah it'll have to be something along those lines.